mlflow/langchain/__init__.py

```diff
@@ -84,6 +86,17 @@
 
 FLAVOR_NAME = "langchain"
 _MODEL_TYPE_KEY = "model_type"
+_DATABRICKS_RAG_CONFIG = "databricks_rag_config"
+__chain__ = None
+__config_path__ = None
+
+
+def _set_chain(chain):
+    globals()["__chain__"] = chain
```

Comment:
This isn't used. Should it be here, or should it be defined in another module like `databricks.rag`?

Reply:
We need that users in MLflow folks can do that and also from databricks.rag we import mlflow.langchain and set that as well. So its reused for that.

Reply:
This is only for Databricks RAG, not MLflow users. Do we need to put this in mlflow.langchain for the Databricks RAG use case?

Reply:
I think it would be easier for databrick.rag to depend on mlflow.langchain rather than creating a circular dependency. What do you think?

Reply:
I'd rather isolate this funny business to `databricks.rag` for now. It's a bit odd to bleed it into the langchain package.